src/CommentSection.module.css

```diff
@@ -1,7 +1,3 @@
-a {
-  text-decoration: underline;
-}
```

Comment:
@jasonday I had to change this because the built css was overriding the a tags on the whole page instead of just the comment section. let me know if you see any issues with the update.